viewer/src/app/feature-service.service.ts

```diff
@@ -108,10 +143,13 @@ export class FeatureServiceService {
   constructor(private http: HttpClient) {}
 
   getFeatures(url: DataUrl): Observable<Feature<Geometry>[]> {
-    console.log(url)
+    console.log(JSON.stringify(url))
     return this.http.get<featureCollectionGeoJSON>(url.url).pipe(
       map(data => {
-        return new GeoJSON().readFeatures(data, { featureProjection: url.projection })
+        return new GeoJSON().readFeatures(data, {
+          dataProjection: url.dataMapping.dataProjection,
+          featureProjection: url.dataMapping.visualProjection,
```

Comment:
featureProjection of visualProjection? Features zijn data dus neem aan dat dit visualProjection moet heten?

Reply:
zie https://openlayers.org/en/latest/apidoc/module-ol_format_GeoJSON-GeoJSON.html#readFeatures